lms/djangoapps/courseware/views.py

```diff
@@ -638,9 +638,23 @@ def jump_to(_request, course_id, location):
     elif section is None:
         return redirect('courseware_chapter', course_id=course_key.to_deprecated_string(), chapter=chapter)
     elif position is None:
-        return redirect('courseware_section', course_id=course_key.to_deprecated_string(), chapter=chapter, section=section)
+        return redirect(
+            'courseware_section',
+            course_id=course_key.to_deprecated_string(),
```

Comment:
@cpennington, @mattdrayer Changed the `jump_to` function to use all non-deprecated methods (Seemed silly to fix the `to_deprecated_string` calls and not fix the corresponding `from_deprecated_string` calls) - tests are passing locally, and I imagine that everything will likewise work on jenkins... waiting.